web/libs/ui/src/lib/data-table/data-table.tsx

```diff
@@ -111,6 +111,33 @@ export const DataTable = <T extends DataShape>(props: DataTableProps<T>) => {
   const [internalSorting, setInternalSorting] = useState<SortingState>([]);
   const [internalActiveRowId, setInternalActiveRowId] = useState<string | undefined>(undefined);
 
+  // Restore column sizes from localStorage if storageKey is provided
+  const restoredColumnSizing = useMemo(() => {
+    if (!props.cellSizesStorageKey) return {};
+
+    try {
+      const stored = localStorage.getItem(props.cellSizesStorageKey);
```

Comment:
I didn't see setItem anywhere in the PRs, is that missing or was it implemented previously and we weren't reading from it yet?